### ISSUE TITLE
Include pass/fail for WCAG 1.4.11 Non-text contrast

### ISSUE THREAD

**Issue by lloydi** [superhighfives/pika#30]
Simple one really - where you have the pass/fail inidicators for text contrast, it would be good to have another row detailing pass/fail for [SC 1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast) (Perhaps this would be enabled in prefs)

**Comment by superhighfives** [comment]
Hmm, given this is the same contrast as AA large, maybe it would be easier to simply edit the copy for AA+? 

![A screenshot of Pika showing the help text for AA+ contrast](https://user-images.githubusercontent.com/449385/106212826-5525f500-61c3-11eb-9185-79ef45775b36.png)

For example, could update it from:

```diff
- let AAPlusText = "WCAG 2 level AA requires a contrast ratio of at least 3:1 for bold or large text."
+ let AAPlusText = "WCAG 2 level AA requires a contrast ratio of at least 3:1 for bold or large text, and for non-text content like UI."
```

I've got an issue to better explore the iconography for the AA / AA+ / AAA / AAA+, too, which this could be a part of. https://github.com/superhighfives/pika/issues/15
